nixos/tests/buildbot.nix

```diff
@@ -3,10 +3,7 @@
   pkgs ? import ../.. { inherit system config; }
 }:
 
-with import ../lib/testing.nix { inherit system pkgs; };
-
-# Test ensures buildbot master comes up correctly and workers can connect
```

Comment:
I appreciate comments, could you readd it at the top of the file?

Reply:
Done